### ISSUE TITLE
Image tags should have alt text

### ISSUE THREAD

**Issue by nellarro** [dontbecypher16/Top-5#3]
🛠️ All images should have alternative text. 

Alt text is [super important](https://moz.com/learn/seo/alt-text). It helps just in case the image is no longer available as well as helps if a user is not a sighted user.
